numpy/linalg/linalg.py

```diff
@@ -2545,7 +2545,7 @@ def norm(x, ord=None, axis=None, keepdims=False):
         elif ord == 1:
             # special case for speedup
             return add.reduce(abs(x), axis=axis, keepdims=keepdims)
-        elif ord is None or ord == 2:
+        elif (ord is None or ord == 2 or ord in ('f', 'fro')):
```

Comment:
It's tempting to handle this on line 2538, as:
```
# frobenius is equivalent to the 2-norm, which is the default
if ord is None or ord in ('f', 'fro'):
    ord = 2
```
which makes this line just `ord == 2`

Reply:
I think that would work as well. I am unsure which would be the superior option. I am rather new to submitting pull requests so I might defer to your judgment or more feedback.

Reply:
I agree with @eric-wieser - organizing it in this way makes explicit the relationship with the default behavior

Reply:
I agree with @eric-wieser - his proposed approach is more explicit in demonstrating the relationship between the default behavior (i.e. `ord=None`) and `ord='fro'`.

Reply:
While we're at it - line 2553 is difficult to grok and seems to be the exception raising mechanism for `ord='fro'` for 1D vectors. It should probably be cleaned up to be consistent with the other changes in this PR.